Makefile

```diff
@@ -56,7 +56,7 @@ $(KUSTOMIZE): # Build kustomize from tools folder.
 	$(MAKE) -C $(ROOT) kustomize
 
 # Define Docker related variables. Releases should modify and double check these vars.
-REGISTRY ?= localhost:5000
+REGISTRY ?= 127.0.0.1:5000
```

Comment:
This fixes an issue with operating systems that translate dns names to ipv6.

Reply:
This is fixing issue @isaacdorfman has on his computer
Is it ok changing it here in order to avoid similar errors in the future?
@davidvossel @agradouski @cchengleo @nunnatsa  ^

Reply:
i think it's likely fine

Reply:
I think this will be okay as we always specify the REGISTRY env variable.

Reply:
should be fine